packages/yoshi-flow-editor/src/model.ts

```diff
@@ -60,14 +71,30 @@ export async function generateFlowEditorModel(
       const pageFileName = resolveFrom(componentDirectory, 'Page');
       const controllerFileName = resolveFrom(componentDirectory, 'controller');
       const settingsFileName = resolveFrom(componentDirectory, 'Settings');
+      const configFileName = resolveFrom(componentDirectory, '.component');
+      const componentConfig =
+        configFileName && getComponentConfig(configFileName);
+      const componentPathRelativeToRoot = path.relative(
+        path.resolve('.'),
```

Comment:
Not critical, but I find using `process.cwd()` a bit clearer than `path.resolve('.')` 
```suggestion
        process.cwd(),
```